### PR TITLE
925953 - [WinUI Kanban] Added swimlane content changes

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description
Added swimlane content changes for WinUI Kanban control.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -375,6 +375,7 @@
         <ul>
             <li> <a href="/winui/Kanban/overview">Overview</a></li>
             <li><a href="/winui/Kanban/getting-started">Getting Started</a></li>
+            <li><a href="/winui/Kanban/swimlane">Swimlane</a></li>
```
Comment (Karthickmani97) [inline winui-toc.html:378]: Follow this order

- Overview
- Getting Started
- Columns
- Cards
- Swimlane

File: winui/Kanban/swimlane.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swim lanes in WinUI Kanban Control
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:10]: Swimlane in WinUI Kanban (SfKanban) control

File: winui/Kanban/swimlane.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+The Swim lanes organize the Kanban board horizontally, allowing you to categorize cards by key attributes such as assignee, project, or task type.
+
+By default, swimlanes are categorized based on the `Assignee` property in the `KanbanModel` class. However, you can customize the categorization by mapping the `SwimlaneKey` property to the appropriate property name in `KanbanModel`.
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:14]: By default, swimlanes are categorized based on the `Assignee` values in the `KanbanModel` class. You can also define category by mapping `SwimlaneKey` to appropriate property name in the defined kanban model.

File: winui/Kanban/swimlane.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property:
+
+{% tabs %}
+{% highlight XAML tabtitle="MainWindow.xaml" hl_lines="2" %}
+
+<kanban:SfKanban x:Name="kanban"
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:21]: Add XAML, C# and ViewModel as sub titles

File: winui/Kanban/swimlane.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# tabtitle="MainWindow.xaml.cs" %}
+
+this.kanban.SwimlaneKey = "IndicatorColorKey";
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:33]: Add all properties, as used in XAML and highlight SwimlaneLey

File: winui/Kanban/swimlane.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+        return taskDetails;
+    }
+
+    #endregion
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize swim lane header
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:165]: Header appearance customization

File: winui/Kanban/swimlane.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+
+The WinUI Kanban control allows you to customize the appearance of swimlane headers using the SwimlaneHeaderTemplate property. This enables you to create a unique design for the header displayed at the start of each swimlane group.
+
+The following code example demonstrates how to customize a swimlane header:
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:169]: The following code example demonstrates how to customize a swimlane header.

File: winui/Kanban/swimlane.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+{% highlight XAML tabtitle="MainWindow.xaml" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 SwimlaneKey="IndicatorColorKey">
+    <kanban:SfKanban.SwimlaneHeaderTemplate>
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:176]: Highlight all Headertemplate code

File: winui/Kanban/swimlane.md [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+        <DataTemplate x:Name="swim">
+            <Grid>
+                <Border BorderBrush="Black" CornerRadius="5,5,5,5" Width="150" Margin="10,2,10,0" HorizontalAlignment="Left" >
+                    <StackPanel Background="LightGray" x:Name="SwimlaneHeaderPanel" Orientation="Horizontal">
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:180]: add C# code, set Itemsource, Swimkey value alone.

File: winui/Kanban/swimlane.md [hunk 10]
```diff
@@ -0,0 +1,220 @@
...
+                               VerticalAlignment="Center" Text="{Binding Title}" />
+                </StackPanel>
+                </Border>
+            </Grid>
+        </DataTemplate>
+    </kanban:SfKanban.SwimlaneHeaderTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:218]: Add notes:

The `DataContext` for the `SwimlaneHeaderTemplate` property in `SfKanban` is set to the `KanbanModel`.

File: winui/Kanban/swimlane.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:10]: Changed.

File: winui/Kanban/swimlane.md [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swim lanes in WinUI Kanban Control
+
+The Swim lanes organize the Kanban board horizontally, allowing you to categorize cards by key attributes such as assignee, project, or task type.
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:12]: Changed.

File: winui/Kanban/swimlane.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:14]: Changed.

File: winui/Kanban/swimlane.md [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+
+By default, swimlanes are categorized based on the `Assignee` property in the `KanbanModel` class. However, you can customize the categorization by mapping the `SwimlaneKey` property to the appropriate property name in `KanbanModel`.
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property:
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:16]: Changed.

File: winui/Kanban/swimlane.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:165]: Changed.

File: winui/Kanban/swimlane.md [hunk 13]
```diff
@@ -0,0 +1,220 @@
...
+
+    #endregion
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize swim lane header
+
+The WinUI Kanban control allows you to customize the appearance of swimlane headers using the SwimlaneHeaderTemplate property. This enables you to create a unique design for the header displayed at the start of each swimlane group.
```
[the remark this answers is not in the document]
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:167]: Changed.

File: winui/Kanban/swimlane.md (hunk 7 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:169]: Changed.

File: winui/Kanban/swimlane.md (hunk 10 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:218]: Added.

File: winui/Kanban/swimlane.md (hunk 4 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:21]: Added.

File: winui/Kanban/swimlane.md (hunk 5 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:33]: Added.

File: winui/Kanban/swimlane.md (hunk 8 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:176]: Changed.

File: winui/Kanban/swimlane.md (hunk 9 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:180]: Added.

File: winui-toc.html (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply winui-toc.html:378]: Will add once Columns and Cards merged.

File: winui/Kanban/swimlane.md [hunk 14]
```diff
@@ -0,0 +1,359 @@
...
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swimlane in WinUI Kanban (SfKanban) control
+
+Swimlanes are horizontal categorizations of cards on the WinUI Kanban Board. It is used for grouping of cards, which brings transparency to the workflow process They allow you to categorize your workflow by projects, teams, users, or any other criteria you need.
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:12]: Swimlanes are horizontal categorizations of cards on the WinUI Kanban Board. It is used for grouping of cards, which brings transparency to the workflow process. They allow you to categorize your workflow by projects, teams, users, or any other criteria you need.

File: winui/Kanban/swimlane.md [hunk 15]
```diff
@@ -0,0 +1,359 @@
...
+By default, swimlanes are categorized based on the `Assignee` values in the `KanbanModel` class. You can also define a category by mapping `SwimlaneKey` to the appropriate property name in the defined `KanbanModel`.
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                    ItemsSource="{Binding TaskDetails}"
+                    SwimlaneKey="IndicatorColorKey">
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:23]: SwimlaneKey = ""
ItemsSource =

Follow this order and change highlight value basedon changes

File: winui/Kanban/swimlane.md [hunk 16]
```diff
@@ -0,0 +1,359 @@
...
+    #endregion
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize swimlane header
+
+The WinUI Kanban control allows you to customize the appearance of swimlane headers using the `SwimlaneHeaderTemplate` property. This enables you to create a unique design for the header displayed at the beginning of each swimlane group.
+
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:169]: The following code example demonstrates, how to customize a swimlane header.

File: winui/Kanban/swimlane.md [hunk 17]
```diff
@@ -0,0 +1,359 @@
...
+documentation: ug
+---
+
+# Swimlane in WinUI Kanban (SfKanban) control
+
+Swimlanes are horizontal categorizations of cards on the WinUI Kanban Board. It is used for grouping of cards, which brings transparency to the workflow process They allow you to categorize your workflow by projects, teams, users, or any other criteria you need.
+
+By default, swimlanes are categorized based on the `Assignee` values in the `KanbanModel` class. You can also define a category by mapping `SwimlaneKey` to the appropriate property name in the defined `KanbanModel`.
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property.
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:16]: The following code example demonstrates, how to group the underlying data collection using the `SwimlaneKey` property.

File: winui/Kanban/swimlane.md [hunk 18]
```diff
@@ -0,0 +1,359 @@
...
+        taskDetail.Category = "Testing";
+        taskDetail.IndicatorColorKey = "Normal";
+        taskDetail.Image = new Image
+        {
+            Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+        };
+
+        taskDetails.Add(taskDetail);
+        return taskDetails;
+    }
```
Comment (Karthickmani97) [inline winui/Kanban/swimlane.md:351]: Add line

File: winui/Kanban/swimlane.md (hunk 14 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:12]: Updated.

File: winui/Kanban/swimlane.md (hunk 15 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:23]: Changed.

File: winui/Kanban/swimlane.md (hunk 17 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:16]: Updated.

File: winui/Kanban/swimlane.md (hunk 18 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:351]: Added.

File: winui/Kanban/swimlane.md (hunk 16 above)
Comment (TamilarasanGunasekaran) [reply winui/Kanban/swimlane.md:169]: Changed.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**